onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt

```diff
@@ -47,4 +49,24 @@ class OnlineJsonApiAuthenticatedTest {
         assertNotNull(runAttempt)
         assertTrue(runAttempt!!.id == "80179")
     }
+
+    @Test
+    fun `GET wishlist`() {
+        //89 Course is C++ Fundamentals Course
+        val wishlistAdd = runBlocking { CBOnlineLib.onlineV2JsonApi.addWishlist(Wishlist(Course("89"))).body() }
+        assertNotNull(wishlistAdd)
+        assertTrue(wishlistAdd?.course?.id=="89")
+
+        val wishlistGet = runBlocking { CBOnlineLib.onlineV2JsonApi.getWishlist().body()?.get() }
+        assertNotNull(wishlistGet)
+        assertTrue(wishlistGet!!.isNotEmpty())
+
+        val wishlistCheck = runBlocking { CBOnlineLib.onlineV2JsonApi.checkIfWishlisted("89").body() }
+        assertNotNull(wishlistCheck)
+        assertTrue(wishlistCheck!=null)
```

Comment:
both assertions are same you know that right? then what's the point of `assertTrue` please use something more informative

Reply:
The only thing this api returns is the new wishlist id, so checking if it is null or not

Reply:
Tell me one thing what's the difference between  `assertNotNull(wishlistCheck)` and `assertTrue(wishlistCheck!=null)`